package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "hawkular-opentracing",
+  "version": "0.0.1",
+  "description": "Hawkular OpenTracing implementation",
+  "keywords": [
+    "opentracing",
+    "distributed tracing",
+    "tracing",
+    "hawkular",
+    "application performance"
+  ],
+  "author": "Pavol Loffay",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/hawkular/hawkular-apm#readme",
```

Comment:
There is no `readme` yet, and you probably don't really need it anyway for this property. Just pointing to the repo URL should be sufficient.